docs/roles/integrator/exchange-integration.md

```diff
@@ -773,9 +841,1104 @@ and we can find its outcome in `receipts_outcome`:
+          },
+          "tokens_burnt": "0"
+        },
+        "proof": [
+          {
+            "direction": "Left",
+            "hash": "1uVJZ8vNQpHMwPA38DYJjk9PpvnHDhsDcMxrTXcwf1s"
+          }
+        ]
+      }
+    ],
+    "status": {
+      "SuccessValue": "eyJ0b3RhbCI6IjEyNTAwMDAwMDAwMDAwMDAwMDAwMDAiLCJhdmFpbGFibGUiOiIwIn0="
+    },
+    "transaction": {
+      "actions": [
+        {
+          "FunctionCall": {
+            "args": "e30=",
+            "deposit": "125000000000000000000000",
+            "gas": 100000000000000,
+            "method_name": "storage_deposit"
+          }
+        }
+      ],
+      "hash": "6sDUF1f5hebpybUbipNJer5ez13EY4HW1VBJEBqZjCEm",
+      "nonce": 47589658000011,
+      "public_key": "ed25519:5UfEFyve3RdqKkWtALMreA9jzsAGDgCtwEXGNtkGeruN",
+      "receiver_id": "ft.demo.testnet",
+      "signature": "ed25519:31PfuinsVvM1o2CmiZbSguFkZKYqAtkf5PHerfexhDbC3SsJWDzRFBpoUYTNDJddhKeqs93GQ3SHtUyqaSYhhQ9X",
+      "signer_id": "serhii.testnet"
+    },
+    "transaction_outcome": {
+      "block_hash": "5XiuxzTpyw6p2NTD5AqrZYNW7SHvpj8MhUCyn1x2KSQR",
+      "id": "6sDUF1f5hebpybUbipNJer5ez13EY4HW1VBJEBqZjCEm",
+      "outcome": {
+        "executor_id": "serhii.testnet",
+        "gas_burnt": 2427959010878,
+        "logs": [],
+        "receipt_ids": [
+          "4urgFabknn1myZkjTYdb1BFSoEimP21k9smCUWoSggA7"
+        ],
+        "status": {
+          "SuccessReceiptId": "4urgFabknn1myZkjTYdb1BFSoEimP21k9smCUWoSggA7"
+        },
+        "tokens_burnt": "242795901087800000000"
+      },
+      "proof": []
+    }
+  }
+}
+```
+</details>
+
+Transfer the tokens:
+  - using NEAR CLI:
+
+    ```bash
+    near call <contract_account_id> ft_transfer '{"receiver_id": "<receiver_account_id>", "amount": "1"}' --accountId <sender_account_id> --amount 0.000000000000000000000001
+    ```
+
+    Result example:
+
+    ```bash
+    Scheduling a call: berryclub.ek.near.ft_transfer({"receiver_id": "volovyk.near", "amount": "1"}) with attached 0.000000000000000000000001 NEAR
+    Receipt: GDeE3Kv1JHgs71A22NEUbgq55r2Hvcnis8gCMyJtQ2mx
+    	Log [berryclub.ek.near]: Transfer 1 from serhii.near to volovyk.near
+    Transaction Id 3MkWKbXVP8wyy4pBofELqiE1pwx7ie2v3SKCwaobNcEe
+    To see the transaction in the transaction explorer, please open this url in your browser
+    https://explorer.mainnet.near.org/transactions/3MkWKbXVP8wyy4pBofELqiE1pwx7ie2v3SKCwaobNcEe
+    ''
+    ```
+
+  - with JSON RPC call:
+
+      ```bash
+      http post https://rpc.testnet.near.org jsonrpc=2.0 id=dontcare method=broadcast_tx_commit \
+          params:='["CwAAAHNlcmhpaS5uZWFyAAmQpgZcJM5nMc6f3tqmw/YI4eAvc84ZgsKMRRRzhY/6CQAAAAAAAAARAAAAYmVycnljbHViLmVrLm5lYXLLWPIiUOElkDF3u4hLAMJ0Sjeo1V338pDdHIp70va3ewEAAAACCwAAAGZ0X3RyYW5zZmVyKwAAAHsicmVjZWl2ZXJfaWQiOiJ2b2xvdnlrLm5lYXIiLCJhbW91bnQiOiIxIn0AQHoQ81oAAAEAAAAAAAAAAAAAAAAAAAAA7fDOZQt3zCtdS05Y8XaZFlwO/Gd5wkkNAHShzDiLQXk4Q4ixpraLPMJivs35PZD0gocXl1iGFbQ46NG3VllzCA=="]'
+      ```
```

Comment:
@mikedotexe I fear this is not helpful as is. I had an idea in my backlog to write the docs about transaction construction, but so far it has not happened. I wonder how you came up with this transaction string... :thinking: 

I am using my `near-cli` (Rust implementation) to construct a transaction. Here is a completely offline version of the command (it does not require any network/internet connection and no other tool but `near-cli` binary itself):

```
$ near-cli construct-transaction \
    offline --block-hash 68GkSsoBuDMEHxELV3k3kqb7z6N8ij11Mj9XX4Wv97EV --nonce 123 \
    sender serhii.near \
    receiver berryclub.ek.near \
    add-action call-function --method-name ft_transfer --args '{"receiver_id": "volovyk.near", "amount": "1"}' --deposit '0 NEAR' --gas '100 TGas' skip \
    sign-private-key \
      --signer-secret-key 'ed25519:5Aj1vumfqLGNQjwsQD93fDzKZmhvBdAyCBkwrC83aKMqdyLHdcVyAFZ4W3Ej7Y4dsFPKNiVM99wfaSZReJSq5qz7' \
      --signer-public-key 'ed25519:8AJzPPSmGpNzGCUqsvz4X8zHomUEMqCM27ihgkKjQysX' \
    display
```

Reply:
Gotcha, so the way we were dealing with this is to have links to a page in docs where you create transactions.

>To follow this guide, please check the step by step instructions on how to create a transaction first.

Which links to:
https://docs.near.org/docs/tutorials/create-transactions#low-level----create-a-transaction

What do you think?

Reply:
Yeah, providing the link definitely makes sense. Maybe we want to mentions some details about the transaction that needs to be sent in this particular case, which is:

```
Transaction {
        block_hash: `68GkSsoBuDMEHxELV3k3kqb7z6N8ij11Mj9XX4Wv97EV`,
        signer_id: "serhii.near",
        public_key: ed25519:3oN25HE9tf9YUGXQyXCFB2rEtpomijG8ghVV2JvrA4zJ,
        nonce: 123,
        receiver_id: "berryclub.ek.near",
        actions: [
            FunctionCall(
                FunctionCallAction {
                    method_name: ft_transfer,
                    args: `{"receiver_id": "volovyk.near", "amount": "1"}`,
                    gas: 100000000000000,
                    deposit: 0,
                },
            ),
        ],
    }
```

You may format it as JSON and only highlight `signer_id`, `receiver_id`, and `actions`.

Reply:
Thanks, Frol, I've pushed up changes with this idea in mind. I think it's a great idea.